src/types/drupal/media.ts

```diff
@@ -4,13 +4,23 @@ import { DrupalMedia, DrupalFile } from 'next-drupal'
 export const enum MediaResourceType {
   Image = 'media--image',
   Document = 'media--document',
+  Video = 'media--video',
 }
 
 export interface DrupalMediaImage extends DrupalMedia {
   field_description: string
   image: DrupalFile
 }
 
+interface UpdatedDrupalFile extends Omit<DrupalFile, 'drupal_internal__fid'| 'drupal_internal__mid'> {
```

Comment:
add a comment here explaining why this is necessary (underlying bug in next-drupal types)